src/components/dashboard/menu-table/popup-forms.js

```diff
@@ -94,7 +94,84 @@ const StyledTagsForm = styled.div`
     }
 `
 
-const TagsForm = ({ tags, setTags }) => {
+const DishFormTitle = styled.div`
+    position: static;
+    width: 800px;
+    
+    left: 20px;
+    top: 20px;
+    font-family: HK Grotesk Bold;
+    // font-style: normal;
+    //font-weight: bold;
+    font-size: 28px;
+    line-height: 34px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.02em;
+    color: #000000;
+    
+    align-self: flex-start;
+`
+const DishFormSubtitle = styled.div`
+    position: static;
+    width: 800px;
+    height: 12px;
+    left: 20px;
+    font-family: HK Grotesk Regular;
+    
+    font-size: 10px;
+    line-height: 12px;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    color: #000000;
+    margin-top: 10px;
+`
+
+const NewDivider = styled.div`
+    border-bottom: 3px solid black;
+    border:none;
+    display: block;
+    position: absolute;
+    content: '';
+    left: 0;
+    bottom: 0;
+    right: auto;
+    top: auto;
+    width: 100%;
+`
+
+const Divider = ({ color }) => (
+    <hr
+        style={{
+            borderColor: color,
+            width: "100%",
+        }}
+    />
+);
+
+let StyledButton = styled.button`
+    display: block;
+    margin: 20px 0;
+    font-size: 18px;
+    line-height: 22px;
+    color: ${ props => props.theme === 'light' ? "#F3A35C" : "white"};
+    padding: 10px 86px;
+    background: ${ props => props.theme === 'light' ? "white" : "#F3A35C"};
+    border-radius: 8px;
+    border: 2px solid #F3A35C;
+    transition: 0.3s ease-in-out;
+    font-family: HK Grotesk;
+font-size:14px;
+    &:hover {
+        background: rgba(242, 153, 74, 0.2);
+    }
+`
+
+const FormButton = (props) => (
+    <StyledButton theme={props.theme} onClick={props.onClick}>{props.text}</StyledButton>
+)
+
+const TagsForm = ({ tags, setTags, existTags }) => {
     const [allTags, setAllTags] = useState([])
     const [selectedTags, setSelectedTags] = useState([])
```

Comment:
Is selectedTags redundant? I appears to be exactly the same & in-sync with tags/setTags. Looks like we can remove this usestate.